src/core/annotation.js

```diff
@@ -705,6 +705,11 @@ class Annotation {
       this.data.pageIndex = params.pageIndex;
     }
 
+    const it = dict.get("IT");
+    if (it) {
```

Comment:
This isn't enough validation as-is.
```suggestion
    if (it instanceof Name) {
```